config/src/parameters/user.rs

```diff
@@ -512,21 +512,21 @@ impl ChainWide {
 pub struct Torii {
     #[config(env = "API_ADDRESS")]
     pub address: WithOrigin<SocketAddr>,
-    #[config(default = "defaults::torii::MAX_CONTENT_LENGTH.into()")]
-    pub max_content_length: HumanBytes<u64>,
+    #[config(default = "defaults::torii::MAX_CONTENT_LENGTH")]
+    pub max_content_length_bytes: Bytes<u64>,
```

Comment:
```suggestion
    pub max_content_len: Bytes<u64>,
```

I don't think a suffix is needed here

Reply:
@nxsaken what do you think? Just as an example rust's std has `string.len()` and returns len in bytes

Reply:
Generally agree, if the type conveys that info, then the field name should stay minimal IMO. But these fields seem to get de/serialized into an untyped format, so maybe they should contain the full info.

Reply:
I would assume that length is always in bytes or we could have that as the default assumption